code/game/objects/items/weapons/teleportation.dm

```diff
@@ -185,3 +190,67 @@ Frequency:
 /obj/item/weapon/hand_tele/attackby(obj/item/C, mob/living/user)
 	if(istype(C, suitable_cell) && !cell && insert_item(C, user))
 		src.cell = C
+
+
+///////////////////////////////////////
+////////////HANDMADE TELE-STUFF////////
+///////////////////////////////////////
+
+/obj/item/weapon/hand_tele/handmade
+	name = "Handmade hand-teleporter"
+	desc = "Handmade version of hand-tele. Woah, that's was they call an experimental science!"
+	icon_state = "hm_hand-tele"
+	portal_type = /obj/effect/portal/unstable
+	portal_fail_chance = 50
+	cell_charge_per_attempt = 50
+	var/calibration_requered = TRUE
+
+/obj/item/weapon/hand_tele/handmade/attackby(obj/item/C, mob/living/user)
+	..()
+	if(istype(C, /obj/item/weapon/tool/screwdriver))
+		if(user.a_intent == I_HURT)
+			if(prob(5))
+				var/turf/teleport_location = pick( getcircle(user.loc, 3) )
+				user.drop_from_inventory(user.get_active_hand())
+				user.drop_from_inventory(user.get_inactive_hand())
+				if(teleport_location)
+					do_teleport(user, teleport_location, 1)
+					return
+			if(do_after(user, 30))
+				if(calibration_requered)
+					user << SPAN_WARNING("You roughly moving with screwdriver inside of [src], making it unreliable.")
+					portal_fail_chance = 90
+				else
+					calibration_requered = TRUE
```

Comment:
Change this to calibration_requered = FALSE

Reply:
Same here. And there calibration_requered is already false
We recalibrate it to default, so here we change it to true